### DISCUSSION TITLE
Find videos that allow monetization

### DISCUSSION THREAD

**Discussion by inetmarketeers** [Tyrrrz/YoutubeExplode#563] [Q&A]
Is this something that the YouTube explode can handle?

**Comment by Tyrrrz** [comment]
Nope

**Reply by inetmarketeers** [reply]
Do you know any other c# tools that do? Apart from the YT API I mean.

**Reply by Tyrrrz** [reply]
I don't think any exists. I'm not sure if this information is even available through the API.
